tests/system/alerts/test_alerts.py

```diff
@@ -188,7 +193,8 @@ def _generate_alerts(
         ]
 
         for alert_kind in alerts_kind_to_test:
-            alert_name = f"drift_webhook_{alert_kind.value}"
+            normalized_kind = self._normalize_alert_name(alert_kind)
+            alert_name = f"drift-webhook-{normalized_kind}"
```

Comment:
since the method is `normalize` name, it looks weird that you pass the kind.
You can do it all at once:
```suggestion
            alert_name = mlrun.utils.helpers.normalize_name(f"drift-webhook-{alert_kind}")
```